backend/tests/Feature/Auth/LogoutActionTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Actions\Auth;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+/**
+ * @internal
+ *
+ * @coversNothing
+ */
+final class LogoutActionTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testログアウトできる(): void
+    {
+        $user = User::factory()->create(['email' => 'test@example.com']);
+
+        $this->actingAs($user)
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter #1 $user of method Illuminate\Foundation\Testing\TestCase::actingAs() expects Illuminate\Contracts\Auth\Authenticatable, Illuminate\Database\Eloquent\Model given.